invenio_app_ils/config.py

```diff
@@ -219,6 +222,10 @@ def _(x):
         "task": "invenio_accounts.tasks.clean_session_table",
         "schedule": timedelta(minutes=60),
     },
+    "overdue_loans": {
+        "task": "invenio_app_ils.circulation.mail.tasks.send_auto_overdue_mail",
```

Comment:
I would suggest to be slightly more specific in the naming mentioning `loan`, `send_overdue_loans_mail_reminder`...